docs/developer.md

```diff
@@ -31,39 +31,12 @@ You can use fake-executor
  
 This is recommended when working on features that are purely Armada specific or if you want to get a high load of jobs running through Armada components 
 
-#### Setup Kind development
+#### Setup Kind + Docker Compose development
 
-1. Get kind (Installation help [here](https://kind.sigs.k8s.io/docs/user/quick-start/))
+1. Start the "localdev" docker-compose environment
```

Comment:
Should we keep the kind installation instructions?  

Or does your script install it for you?